common/static/common/js/discussion/utils.js

```diff
@@ -321,7 +321,8 @@
             imageUploadUrl = this.urlFor('upload');
             _processor = function(self) {
                 return function(text) {
-                    return self.postMathJaxProcessor(text);
+                    // HTML returned by Markdown is assumed to be safe to render
+                    return self.postMathJaxProcessor(edx.HtmlUtils.HTML(text)).toString();
```

Comment:
Why do you now need to do a ``toString`` on the results of ``postMathJaxProcessor``? That part didn't change, I would have thought.